source/playbooks/work-with-runs.rst

```diff
@@ -0,0 +1,41 @@
+Work with runs
+==============
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+A run is the execution of a playbook. You can start each run in a new channel or you can elect to use the same channel for multiple runs.
+
+To access runs, select the product menu in the top-left corner of Mattermost, then select **Playbooks**. In the runs list, you can select a run to view more details, such as the overview and retrospective. This is an easy way to assess all the active runs to which you have access.
+
+Follow and participate
+----------------------
+
+You don't have to be in a run's channel to follow the run. You can:
+
+- **Follow**: If you're following a playbook, you won't necessarily be added to each of the playbook's runs but you will be added as a follower. To join a run channel, select **Participate** in the header of that run.
+- **Participate**: If you're participating in a run it's likely because you're in a team or group of people who've been added to it. In this case, you'll be able to follow the run in the run channel and also view the details in the list of runs in the playbooks tab.
```

Comment:
I would explain the difference between being a follower and a participant:
- follower: I want to be notified with status updates and relevant events.
- participant: I want to actively participate in the run, checking tasks, changing status, inviting other members...

Note that to be able to read the progress/status (actively, not through notifications) is not necessary to be follower or a participant.

Don't use my words, just wanted to explain the concept 🙏🏽 



D